### ISSUE TITLE
MaterialDesignThemes.MahApps.FlyoutAssist not accessible via Xaml

### ISSUE THREAD

**Issue by Insire** [ButchersBoy/MaterialDesignInXamlToolkit#490]
Hi,

I'm not able to reference the FlyoutAssist introduced in #121 . 
You can reproduce the issue with this [Repo](https://github.com/Insire/MaterialDesignInXamlToolkitTestApp/blob/master/TestApp/MainWindow.xaml#L29) 

![flyoutassist](https://cloud.githubusercontent.com/assets/11455827/18229856/805fa902-7288-11e6-8a7d-e7d095b66891.png)

Windos 10 Pro x64
Visual Studio 2015 Community

my packages.config
```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="MahApps.Metro" version="1.2.4.0" targetFramework="net452" />
  <package id="MaterialDesignColors" version="1.1.3" targetFramework="net452" />
  <package id="MaterialDesignThemes" version="2.1.0.657" targetFramework="net452" />
  <package id="MaterialDesignThemes.MahApps" version="0.0.6" targetFramework="net452" />
</packages>
```

Is there something i'm doing wrong or missing?

**Comment by mgnslndh** [comment]
I can reproduce this bug. My own use of FlyoutAssist is from code and not XAML so i've never encountered the problem. 

It can be fixed by adding `MaterialDesignThemes.MahApps` to the XAML namespace but i am not sure if @ButchersBoy wants it that way. I could not make it work just using `clr-namespace`. 

I'll submit a PR and we'll see how it goes.

**Comment by ButchersBoy** [comment]
Merged, thanks @mgnslndh

**Comment by Insire** [comment]
confirmed fixed with 0.07
